spread-shellcheck

```diff
@@ -14,106 +14,199 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+import logging
+import os
+import subprocess
+import argparse
+
+import yaml
+
+
 # default shell for shellcheck
-SHELLCHECK_SHELL=${SHELLCHECK_SHELL:-bash}
+SHELLCHECK_SHELL = os.getenv('SHELLCHECK_SHELL', 'bash')
 # set to non-empty to ignore all errors
-NO_FAIL=${NO_FAIL:-}
+NO_FAIL = os.getenv('NO_FAIL', None)
 # set to non empty to enable 'set -x'
-D=${D:-}
+D = os.getenv('D', None)
```

Comment:
I'm slightly unhappy about vars like: `D` or `V`. I would prefer something less short like `SHELLCHECK_DEBUG` and `SHELLCHECK_VERBOSE`